CI/check-jsonschema.py

```diff
@@ -0,0 +1,90 @@
+import json
+from jsonschema import Draft7Validator
+from json_source_map import calculate
+from json_source_map.errors import InvalidInputError
+import os
+import sys
+
+json_data = {}
+json_string = ''
+schema = {}
+
+inputArgs = sys.argv
+errors = []
+
+
+def main():
+    if len(sys.argv) < 2:
+        print("JSON path required.")
+        return 1
+
+    for filename in inputArgs[1:]:
+        prep(filename)
+
+    with open('validation_errors.json', 'w') as outfile:
+        json.dump(errors, outfile)
+
+    if len(errors) > 0:
```

Comment:
Personal taste thing, but `if errors:` works fine as well as `errors` is only truthy if it is non-empty.